src/test/system_tests.cpp

```diff
@@ -27,25 +27,44 @@ BOOST_AUTO_TEST_CASE(dummy)
 
 BOOST_AUTO_TEST_CASE(run_command)
 {
+#ifdef WIN32
+    // https://www.winehq.org/pipermail/wine-devel/2008-September/069387.html
+    auto hntdll = GetModuleHandleA("ntdll.dll");
+    assert(hntdll);
+    const bool wine_runtime = GetProcAddress(hntdll, "wine_get_version");
+#endif
+
     {
         const UniValue result = RunCommandParseJSON("");
         BOOST_CHECK(result.isNull());
     }
     {
+#ifdef WIN32
+        const UniValue result = RunCommandParseJSON("cmd.exe /c echo {\"success\": true}");
```

Comment:
Why is an extra cmd.exe needed?

Reply:
It is not "extra", but required. Consider this output:
```
C:\Users\hebasto>where echo
INFO: Could not find files for the given pattern(s).

C:\Users\hebasto>where cmd
C:\Windows\System32\cmd.exe
```